doc/source/development/developer.rst

```diff
@@ -53,26 +60,42 @@ for each column, *including the index columns*. This has JSON form:
     'numpy_type': numpy_type,
     'metadata': metadata}
 
-.. note::
+See below for the detailed specification for these
+
+Index Metadata Descriptors
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``RangeIndex`` can be stored as metadata only, not requiring serialization. The
+descriptor format for these as is follows:
+
+.. code-block:: python
+
+   {'kind': range,
```

Comment:
Should the `range` here be s string literal?

Reply:
yes, will fix